scripts/handle-user-data

```diff
@@ -30,29 +31,33 @@ mount_partitions() {
         mkdir -p /var/lib
     fi
 
-    mkdir -p /mnt/$PARTNAME/var/lib/docker
-    ln -s /mnt/$PARTNAME/var/lib/docker /var/lib/docker
+    mkdir -p $MOUNTPOINT_DATA/var/lib/docker
+    ln -s $MOUNTPOINT_DATA/var/lib/docker /var/lib/docker
 
-    mkdir -p /mnt/$PARTNAME/var/lib/boot2docker
-    ln -s /mnt/$PARTNAME/var/lib/boot2docker /var/lib/boot2docker
+    mkdir -p $MOUNTPOINT_DATA/var/lib/boot2docker
+    ln -s $MOUNTPOINT_DATA/var/lib/boot2docker /var/lib/boot2docker
 
-    mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
-    ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
+    mkdir -p $MOUNTPOINT_DATA/var/lib/boot2docker/etc/docker
+    ln -s $MOUNTPOINT_DATA/var/lib/boot2docker/etc/docker /etc/docker
 
-    mkdir -p /mnt/$PARTNAME/var/lib/minishift
-    ln -s /mnt/$PARTNAME/var/lib/minishift /var/lib/minishift
+    mkdir -p $MOUNTPOINT_DATA/var/lib/minishift
+    ln -s $MOUNTPOINT_DATA/var/lib/minishift /var/lib/minishift
 
     # Make sure /tmp is on the disk too
-    rm -rf /mnt/$PARTNAME/tmp || true
-    mv /tmp /mnt/$PARTNAME/tmp
-    ln -fs /mnt/$PARTNAME/tmp /tmp
+    rm -rf $MOUNTPOINT_DATA/tmp || true
+    mv /tmp $MOUNTPOINT_DATA/tmp
+    ln -fs $MOUNTPOINT_DATA/tmp /tmp
 
     # Move userdata to persistent storage
     if [ -e "/userdata.tar" ]; then
         mv /userdata.tar /var/lib/boot2docker/
     fi
 
-    ls -l /mnt/$PARTNAME
+    ls -l $MOUNTPOINT_DATA
+
+    # Setup docker storage
+    echo "VG=minishift_vg" >> /etc/sysconfig/docker-storage-setup
+    docker-storage-setup
```

Comment:
When this has run, it will generate a file /etc/sysconfig/docker-storage. That file is parsed by docker during next run. How do you make sure that file is retained over next reboot?